components/prism-apacheconf.js

```diff
@@ -1,12 +1,12 @@
 		lookbehind: true,
 		alias: 'property'
 	},
 	'directive-block': {
-		pattern: /<\/?\b(?:AuthnProviderAlias|AuthzProviderAlias|Directory|DirectoryMatch|Else|ElseIf|Files|FilesMatch|If|IfDefine|IfModule|IfVersion|Limit|LimitExcept|Location|LocationMatch|Macro|Proxy|RequireAll|RequireAny|RequireNone|VirtualHost)\b *.*>/i,
+		pattern: /<\/?\b(?:Auth[nz]ProviderAlias|Directory|DirectoryMatch|Else|ElseIf|Files|FilesMatch|If|IfDefine|IfModule|IfVersion|Limit|LimitExcept|Location|LocationMatch|Macro|Proxy|Require(?:All|Any|None)|VirtualHost)\b *.*>/i,
```

Comment:
It doesn't look like we have all the `Require` variants in our tests. Can we add them?

Reply:
We do have [these tests](https://github.com/PrismJS/prism/blob/master/tests/languages/apacheconf/directive-block_feature.test#L37-L42).